cpmpy/solvers/pysat.py

```diff
@@ -21,12 +21,11 @@
     Installation
     ============
 
-    Requires that the 'python-sat' package is installed. If you want to also solve
-    pseudo-Boolean constraints, you should also install its optional dependency 'pblib', as follows:
+    Requires that the 'python-sat' package is installed. If you want to also solve pseudo-Boolean and cardinality constraints, you should also install its optional dependency 'pypblib', as follows:
```

Comment:
would be more consistent with other sovlers if

"""
Requires that the 'python-sat' package is installed:

.. code-block:: console
        $ pip install pysat
"""
followed by the 'If you...'
where I think the earlier pip install pysat[pblib] is better because its pysat that chooses the dependent version?

Reply:
Right, I forgot we didn't make this the recommendation in the other solvers here. Plus I can give them the alternative of `pysat`  if `pypblib` fails to compile.